prov/rxd/src/rxd_ep.c

```diff
@@ -664,10 +670,7 @@ void rxd_ep_send_ack(struct rxd_ep *rxd_ep, fi_addr_t peer)
 	ack->ext_hdr.rx_id = rxd_ep->peers[peer].curr_rx_id;
 	rxd_ep->peers[peer].last_tx_ack = ack->base_hdr.seq_no;
 
-	ret = rxd_ep_retry_pkt(rxd_ep, pkt_entry);
-	if (ret)
-		FI_WARN(&rxd_prov, FI_LOG_EP_CTRL, "Unable to send ack\n");
-
+	rxd_ep_retry_pkt(rxd_ep, pkt_entry);
```

Comment:
If the return code isn't used anywhere else, the function can be converted to void